integrated_channels/moodle/client.py

```diff
@@ -143,12 +142,7 @@ def __init__(self, enterprise_configuration):
         """
         super().__init__(enterprise_configuration)
         self.config = apps.get_app_config('moodle')
-        token = (
-            enterprise_configuration.decrypted_token
-            if getattr(settings, 'FEATURES', {}).get('USE_ENCRYPTED_USER_DATA', False)
-            else enterprise_configuration.token
-        )
-        self.token = token or self._get_access_token()
+        self.token = enterprise_configuration.decrypted_token or self._get_access_token()
```

Comment:
Instead of removing this feature flag, let's enable it for prod in `edx-internal` first. Once testing is done on prod, we can then remove this flag and reference the decrypted value directly. 

**Benefits**: If something goes wrong, we can disable the feature flag back in edx-internal that offers speedy deployment within 6-7 minutes. Wherease edx-enterprise takes around 3-4 hours to get deployed.

Reply:
that can be better approach for testing on prod.
@sameenfatima78 updated.